xla/service/gpu/custom_fusion_rewriter.cc

```diff
@@ -166,7 +166,9 @@ static StatusOr<HloInstruction*> CreateFusionInstruction(
   FusionBackendConfig backend_config;
   backend_config.set_kind("__custom_fusion");
   *backend_config.mutable_custom_fusion_config() = match.config();
-  TF_RETURN_IF_ERROR(fusion->set_backend_config(std::move(backend_config)));
+  GpuBackendConfig gpu_config;
```

Comment:
similar but you could just move the `GpuBackendConfig gpu_config;` declaration above, and do

```
FusionBackendConfig& backend_config = *gpu_config.mutable_fusion_backend_config();
```

then drop the assignment by the end. There are other occurrences in other files as well.